news/2023/2023-10-22-halloween-descends.md

```diff
@@ -0,0 +1,121 @@
+
+[![](https://assets.ppy.sh/contests/186/banners/RedcXca.jpg)](https://assets.ppy.sh/contests/186/winners/RedcXca.png)
+
+Fanart contest veteran [**RedcXca**](https://osu.ppy.sh/users/14056601) returns as a finalist once again, and can now proudly claim 'I maid this'. Because they put Yuzu in a maid dress, and we're frankly astonished by how well it works. Alisa in a bunny suit is also present, as well as a guest cameo from the ever-popular Doraemon for reasons we're not even going to pretend to understand, plus several trollface skull decorations. There's more hidden in this piece to boot, but I've got too many *tasks* to do to list them. 
+
+Sure is cloudy tonight, isn't it?
+
+[![](https://assets.ppy.sh/contests/186/banners/oniichan49.jpg)](https://assets.ppy.sh/contests/186/winners/oniichan49.png)
+
+**1210** voters hopped along to [**oniichan49**](https://osu.ppy.sh/users/20871620)'s jiangshi pippi and Aiko grooving out before a floating pumpkin audience. A witchy don is conducting this danse macabre, and we're all here for it... in spirit.
+
+[![](https://assets.ppy.sh/contests/186/banners/Triantafyllia.jpg)](https://assets.ppy.sh/contests/186/winners/Triantafyllia.png)
+
+Decoration is the name of the game in [**Triantafyllia**](https://osu.ppy.sh/users/17084594)'s piece, impressing **1122** voters. So much going on here between pippi and Mocha's incredibly ornate costumes and vibrant artistry really gunning for a beautiful collision of candy-like light and colour.
+
+[![](https://assets.ppy.sh/contests/186/banners/AyoLie.jpg)](https://assets.ppy.sh/contests/186/winners/AyoLie.png)
+
+UHHHHHH BRANES. [**AyoLie**](https://osu.ppy.sh/users/29186645) DREW ZOMBIE PIPPI. ZOMBIE MOCHA TOO. UUUURRRRRR. **1115** VOTERS LIKE VERY MUCH. BRAAAAANES. Also, a cute Chirou's chilling too. We're not entirely sure if she's a disembodied head or not, but we're going to pretend she is, because you know, Halloween. Wonderful piece!
+
+[![](https://assets.ppy.sh/contests/186/banners/Haru34_old.jpg)](https://assets.ppy.sh/contests/186/winners/Haru34_old.png)
+
+**1079** voter ghosts are waiting patiently in line to be hugged by [**Haru34_old**](https://osu.ppy.sh/users/8806199)'s pippi-mummy amidst a spooky violet-tinged moonscape. Why are these spirits so unbearably cute? It is a mystery.
+
+## peppy's Picks
+
+Long ago before time was counted, the Dean "peppy" Herbert once uttered deep within the hallowed crypts (of our internal Discord server) that there were too many good entries and not enough votes to go around. For thousands of years (approximately eleven seconds, we counted) did he contemplate, before deciding that he would pick a selection of his favorite entries to join our finalists on the main menu anyway. We've since discovered that if you gave him triple the number of ordinary votes, he'd still run out of them somehow.
```

Comment:
```suggestion
Long ago before time was counted, the Dean "peppy" Herbert once uttered deep within the hallowed crypts (of our internal Discord server) that there were too many good entries and not enough votes to go around. For thousands of years (approximately eleven seconds, we counted) did he contemplate, before deciding that he would pick a selection of his favourite entries to join our finalists on the main menu anyway. We've since discovered that if you gave him triple the number of ordinary votes, he'd still run out of them somehow.
```